utils/test/garden-test-utils.tsx

```diff
@@ -42,7 +42,6 @@ const customLtrRender = (ui: React.ReactElement, options?: any) =>
 const customRtlRender = (ui: React.ReactElement, options?: any) =>
   render(ui, { wrapper: RtlProvider, ...options });
 
-// eslint-disable-next-line no-duplicate-imports
```

Comment:
If this is the only eslint violation for dupe imports, then the rule should be turned back on in the config.

Reply:
This would affect all type "re-exports" that we eventually add in (similar to the changes in `theming/src/index.ts` and `tabs/src/index.ts`.

We could require these re-export areas to explicitly disable this rule, but it would eventually be in every package.

Reply:
In that case, it would be good to do an `overrides` scoped to those files. Similar to https://github.com/zendeskgarden/website/blob/63979d7ec70c18a75ab5d68a4c5b01939ee4e15e/.eslintrc.json#L51-L59

Reply:
Updated to now be scoped to `src/index.ts` files only